src/App.js

```diff
@@ -1,26 +1,87 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { firebase, db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+const App = () => {
+  const [value, loading, error] = useCollection(
+    firebase.firestore().collection('list'),
+    {
+      snapshotListenOptions: { includeMetadataChanges: true },
+    },
+  );
+
+  console.log(firebase);
+  console.log(db);
+
+  const sendItem = () => {
+    firebase.db
+      .collection('list')
+      .add({ title: 'first item', description: 'new item' })
+      .then((documentReference) => {
+        console.log('document reference ID', documentReference.id);
+      })
+      .catch((error) => {
+        console.log(error.message);
+      });
+  };
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Grocery List</h1>
+      <button onClick={sendItem}>click here to add grocery item</button>
```

Comment:
Good choice in keeping it simple by choosing to use a button to test sending content to the database.